circle.yml

```diff
@@ -12,6 +12,7 @@ checkout:
       fi
 dependencies:
   override:
+    - pip install pip --upgrade
```

Comment:
circleci env has ancient pip. Following the fix for the same issue as encountered by certbot.